java-checks/src/test/files/checks/NullPointerCheck.java

```diff
@@ -345,6 +345,14 @@ public void test(@Nullable Object o) {
     a.hashCode(); // Compliant
   }
 
+  public void testAssignNullableMethod() {
+    Object object;
+    object = nullableMethod();
+    if(object.hashCode()); // issue
```

Comment:
ouch! the  ; for the empty assignement is really less readable than an empty block IMO.